docs/15-mib.md

```diff
@@ -11,49 +11,47 @@ Status
 Database
 ---------------
 
-Traps are held in the database, to get rid of old ones, you can set here the number of days the traps are kept.
+Traps are held in the database. You can set the retention period for old traps here.
 
-This will occur at regular interval (not implemented in beta) or when a trap is received.
+This will occur at regular intervals (not implemented in beta) or when a trap is received.
```

Comment:
```suggestion
This will occur every time a trap is received.
```
Regular purge was planned, but as it takes very little time to execute, it was abandonned. 
If ti comes up again, I will pdate.